app/code/Chandan/Orderlisticon/Block/Order/History.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Chandan\Orderlisticon\Block\Order;
+
+use \Magento\Framework\App\ObjectManager;
+use \Magento\Sales\Model\ResourceModel\Order\CollectionFactoryInterface;
+
+/**
+ * Sales order history block
+ *
+ * @api
+ * @since 100.0.2
+ */
+class History extends \Magento\Sales\Block\Order\History
+{
+    /**
+     * @return void
+     */
+    public function _construct()
```

Comment:
Overriding constructor required ? Try without this

Reply:
I don’t override the constructor because I don’t have to create any custom function in my block file. I have created a function only getTemplate to render the phtml file.